package.json

```diff
@@ -42,7 +42,7 @@
     "@types/jest": "^29.5.7",
     "@vitejs/plugin-vue": "^4.2.3",
     "@vue/test-utils": "^2.4.1",
-    "happy-dom": "^12.10.3",
+    "jsdom": "^22.1.0",
```

Comment:
had to change this because happy dom has a bug which doesn't let you look at css properties (which are required for functions like `isVisible()`)